conduct-response.md

```diff
@@ -0,0 +1,99 @@
+* If you feel people are in immediate physical danger, it can be appropriate to contact local law enforcement.
+* If you witness a presentation which is clearly in violation of the Code of Conduct repeatedly or very seriously. You might simply say “I’m sorry, this presentation cannot be continued at the present time”. Examples of when this is justified could be significant threats of violence, harassment of others, or continious sexist jokes. This should not be done in cases like a few inappropriate jokes - in that case do report it to the CARE team.
+
+When possible, even in these cases, consult with another organiser or volunteer if you see one nearby.
+
+### Guidelines for Code of Conduct response team
+
+The Code of Conduct Active Response Ensurers (CARE) are in charge of handling reports until their completion. It is their task to ensure all reports are handled timely and professionally.
+
+Participants look towards the CARE team for appropriate behaviour, even more than other staff, and that makes it essential for their behaviour to remain within the Code of Conduct. Violations by CARE team members can severely erode trust in the Code of Conduct process.
+
+CARE team members almost never act on their own, including tasks like replying to Code of Conduct related e-mails. If it’s not possible to meet with the entire CARE team in time, at least one other team member should be consulted.
+
+#### Receiving a report
+
+Reports are typically received by e-mail, in person, or from another staff member. If the report was received from someone else, it may be best to meet with the original reporter first, depending on how complete the information is. The information the CARE team aims to collect is listed in the Code of Conduct reporting guide, but sometimes not all information is available.
+
+When receiving a report, the CARE team acknowledges receipt as soon as possible, and aims to be understanding and compassionate. However, there should be no commitment on whether this is a violation or which action will be taken.
+
+#### Acting as a team
+
+The CARE team generally works, decides and communicates as a team. If the report indicates that immediate action is required and other CARE team members are not available, any CARE team member may take the action they see appropriate. The same guidelines for unilateral action apply as listed above for other staff members. If possible, it’s still preferred to take action after a brief discussion with at least one other CARE team member, or even any other organiser, rather than acting entirely unilaterally. Formal processes and guidelines must never get in the way of preventing threats to anyone’s safety.
+
+#### Reviewing the report
+
+Based on the report and any other available information, the CARE team will meet to determine, to the best of their ability:
+
+* what happened
+* whether this event constitutes a code of conduct violation
```

Comment:
Use capitalization `code of conduct` and for first words of the lines `what, wether, who...`